tests/UNMOCKED_end2end_tests/test_api_latency.py

```diff
@@ -0,0 +1,30 @@
+import os
+import time
+
+import numpy as np
+from tqdm import tqdm
+
+from lightly.api import ApiWorkflowClient
+from lightly.openapi_generated.swagger_client import Configuration, ApiClient, QuotaApi
+
+if __name__ == "__main__":
+    token = os.getenv("TOKEN")
+    api_client = ApiWorkflowClient(token=token).api_client
+    quota_api = QuotaApi(api_client)
+
+    no_iters = 200
```

Comment:
I don't want to be picky here but I would use `n_iters` as its commonly done. `no_iters` sounds like `no iterations` but we do 200